Sources/Hyperspace/Request/Request.swift

```diff
@@ -38,10 +38,10 @@ public struct Request<Response, Error: TransportFailureRepresentable>: Recoverab
     /// The creation strategy for generating a `URLRequest`. Can be customized by using the `.custom` case.
     public var urlRequestCreationStrategy: URLRequestCreationStrategy = .default
     
-    /// The maximum number of attempts that this operation should make before completely aborting. This value is nil when there is no maximum.
-    public var maxRecoveryAttempts: UInt?
+    /// The maximum number of attempts this operation should make before completely aborting. A value of nil means there is no maximum.
```

Comment:
Minor: could add backpacks to `nil` for syntax highlighting in the comment.

Reply:
Can change.